package-lock.json

```diff
@@ -0,0 +1,14671 @@
+{
```

Comment:
We need to either go all in on yarn on npm. seems like it will be best if we move everything to npm, but that shouldn't happen in this PR.

Reply:
Let's switch over to npm at a later separate PR, removed the `package-lock.json` for now.